@navikt/core/react/src/process/Process.tsx

```diff
@@ -0,0 +1,396 @@
+
+    const rootId = useId(id);
+
+    const rootRef = useRef<HTMLOListElement>(null);
+    const mergedRef = useMergeRefs(forwardedRef, rootRef);
+
+    const [activeChildId, setActiveChildId] = useState<string | undefined>(
+      undefined,
+    );
+
+    const syncAriaControls = useCallback(() => {
+      const activeChildren = rootRef.current?.querySelectorAll(
+        '[data-process-event][aria-current="true"]',
+      );
+
+      if (!activeChildren) {
+        setActiveChildId(undefined);
+        return;
+      }
+
+      if (activeChildren.length > 1) {
+        if (process.env.NODE_ENV !== "production") {
+          console.warn(
+            "Aksel: Found multiple `<Process.Event />` elements with `status='active'`. Only one event should be active at a time.",
+            rootRef.current,
+          );
+        }
+        setActiveChildId(undefined);
+        return;
+      }
+
+      if (activeChildren.length === 1) {
+        const lastActiveChild = activeChildren[activeChildren.length - 1];
+        setActiveChildId(lastActiveChild.id);
+      } else {
+        setActiveChildId(undefined);
+      }
+    }, []);
+
+    return (
+      <ol
+        ref={mergedRef}
+        data-color="info"
+        role="list"
+        {...restProps}
+        className={cn("navds-process", className)}
+        id={rootId}
+        aria-controls={activeChildId}
+      >
+        <ProcessContextProvider
+          hideStatusLabel={hideStatusLabel}
+          rootId={rootId}
+          syncAriaControls={syncAriaControls}
+        >
+          {children}
+        </ProcessContextProvider>
+      </ol>
+    );
+  },
+) as ProcessComponent;
+
+/* ------------------------------ Process Event ------------------------------ */
+interface ProcessEventProps extends React.HTMLAttributes<HTMLLIElement> {
+  /**
+   * Rich content to display under the title and timestamp if provided.
+   */
+  children?: React.ReactNode;
+  /**
+   * Hide the content section of the event.
+   */
+  hideContent?: boolean;
+  /**
+   * Step title.
+   */
+  title?: string;
+  /**
+   * Timestamp or date to display for event.
+   */
+  timestamp?: string;
+  /**
+   * Icon or number to display inside the bullet.
+   */
+  bullet?: React.ReactNode;
+  /**
+   * Current event status.
+   * @default "inactive"
+   */
+  status?: "active" | "completed" | "inactive";
+}
+
+export const ProcessEvent = forwardRef<HTMLLIElement, ProcessEventProps>(
+  (
+    {
+      title,
+      timestamp,
+      children,
+      bullet,
+      hideContent,
+      className,
+      id,
+      status = "inactive",
+      ...restProps
+    }: ProcessEventProps,
+    forwardedRef,
+  ) => {
+    const translate = useI18n("Process");
+    const { cn } = useRenameCSS();
+    const eventId = useId();
+    const { syncAriaControls, hideStatusLabel, rootId } = useProcessContext();
+
+    const sync = useCallbackRef(syncAriaControls);
+
+    useEffect(() => sync(), [status, sync]);
```

Comment:
I don't think we need to use `useCallbackRef` since `syncAriaControls` is already memoized, and I don't think it needs to be a dep in the `useEffect`.

I think this should suffice:
```suggestion
    useEffect(syncAriaControls, [status]);
```

Reply:
⚡ That's true, changed it and added a comment